helm_deploy/hmpps-education-and-work-plan-api/templates/export-database-to-analytical-platform-cronjob.yaml

```diff
@@ -8,7 +8,8 @@ kind: CronJob
 metadata:
   name: export-database-to-analytical-platform-cronjob
 spec:
-  schedule: "0 9-17 * * 1-5" # On the hour between 09:00 and 17:00 Mon-Fri. For testing only; once smoke tested and working change to every day at 1am -> "0 1 * * *"
+  suspend: true
+  schedule: "0 1 * * *" # 1am every day
```

Comment:
Now that we have tested this in `dev` and seen the data copy across every hour, we can change the cron to once a day.
The real change in this PR however is the `suspend: true` property, which means the cronjob wont run at all. This means that if we have to promote `main` to `preprod` and `prod` soon (which I think we will to deliver other features and bugfixes that we've been working on recently) then there is no risk of this running in `preprod` or `prod`